src/components/layout/Navbar/style.module.scss

```diff
@@ -77,7 +89,86 @@
     background: vars.$wainbow;
     height: 0.25rem;
     position: absolute;
-    top: 4rem;
+    top: 4rem; // Position the wainbow at the bottom
+    transform: translateY(0); // Move wainbow to bottom of navbar by default
+    transition: 0.3s ease-in-out; // Apply transition to transform property
     width: 100vw;
   }
+
+  .mobileNav {
+    align-items: right;
+    background-color: var(--theme-background);
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: center;
+    line-height: 300%;
+    margin-top: -18rem;
+    padding: 0 1rem;
+    padding-bottom: 3rem; // Add padding to the bottom of the mobileNav
+    position: relative;
+    transition: 0.3s ease-in-out;
+    width: 100vw;
+    z-index: -1;
+
+    &.open {
+      margin-top: 0;
+
+      ~ .wainbow {
+        top: auto;
+      }
+    }
+  }
+
+  .mobileNavItem {
+    align-items: center;
+    display: flex;
+    gap: 0.5rem;
+    padding: 0 0.5rem;
+    text-decoration: none;
+  }
+
+  .toggleIcon {
+    background-color: var(--theme-background);
+    height: 20px;
+    margin-right: auto;
+    padding: 0;
+    position: relative;
+    width: 40px;
+
+    .bar1 {
+      background-color: var(--theme-text-on-background-1);
+      border-radius: 2px;
+      height: 5px;
+      position: absolute;
+      top: 1.5px;
+      transition: 0.3s ease-in-out all;
+      width: 40px;
+
+      &.open {
+        transform: rotate(-45deg) translateY(-10px);
+        transform-origin: top right;
```

Comment:
This is kind of hard to describe just in words but the reason that transform-origin was top right on the main website is because this is the top bar and it was right aligned on the page so this line let us keep the top right corner fixed in place. Since our navbar here is left-aligned, we want to keep the top left corner fixed in place so the bars are basically just rotating instead of moving.

![Screenshot 2023-05-08 at 10 35 45 AM](https://user-images.githubusercontent.com/33165426/236892012-6f935174-2b28-4987-9d41-9829f5e69060.jpg)